marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/graph/nodeExpand.sjs

```diff
@@ -177,14 +184,20 @@ if (isConcept) {
       const objectIRI = item.objectIRI.toString();
       let subjectArr = objectIRI.split("/");
       const objectId = subjectArr[subjectArr.length - 1];
-
+      let entityType =   subjectArr[subjectArr.length - 2];
+      let newLabel = getCustomLabel(entityType,  item.docURI);
```

Comment:
Can we call this variabes "customLabel" since they are not "new"?

Reply:
Done, thanks for your input @aniamy